docs/cn/core-services/Caching.md

```diff
@@ -1,259 +1,507 @@
+    -通常，UFS存储旨在相当长一段时间持久存储大量数据。
+-** Alluxio存储**
+    -Alluxio做为一个分布式缓存来管理Alluxio workers本地存储，包括内存。这个在用户应用程序与各种底层存储之间的快速数据层带来的是显著提高的I / O性能。
+    -Alluxio存储主要用于存储热数据，暂态数据，而不是长期持久数据存储。
+    -每个Alluxio节点要管理的存储量和类型由用户配置决定。
+    -即使数据当前不在Alluxio存储中，通过Alluxio连接的UFS​​中的文件仍然
+    对Alluxio客户可见。当客户端尝试读取仅可从UFS获得的文件时数据将被复制到Alluxio存储中。
+
+![Alluxio存储图]({{ '/img/stack.png' | relativize_url }})
+
+Alluxio存储通过将数据存储在计算节点内存中来提高性能。
+Alluxio存储中的数据可以被复制来形成“热”数据，更易于I/O并行操作和使用。
+
+Alluxio中的数据副本独立于UFS中可能已存在的副本。
+Alluxio存储中的数据副本数是由集群活动动态决定的。
+由于Alluxio依赖底层文件存储来存储大部分数据，
+Alluxio不需要保存未使用的数据副本。
+
+Alluxio还支持让系统存储软件可感知的分层存储，使类似L1/L2 CPU缓存一样的数据存储优化成为可能。
 
 ## 配置Alluxio存储
 
+### 单层存储
+
 配置Alluxio存储的最简单方法是使用默认的单层模式。
 
-请注意本文中提到的本地存储以及`mount`这样的术语特指在本地文件系统中挂载，不要与Alluxio底层存储的`mount`概念混为一谈。
+请注意，此部分是讲本地存储，诸如`mount`之类的术语指在本地存储
+文件系统上挂载，不要与Alluxio的外部底层存储的`mount`概念混淆。
 
-默认配置下的Alluxio将为每个worker提供一个ramdisk，并占用一定的百分比的系统总内存。这个ramdisk将被用作分配给每个Alluxio worker的唯一存储介质。
+在启动时，Alluxio将在每个worker节点上发放一个ramdisk并占用一定比例的
+系统的总内存。 此ramdisk将用作分配给每个Alluxio worker的唯一存储介质。
 
-Alluxio存储通过Alluxio的`alluxio-site.properties`配置。详细配置请参考[配置文档]({{ '/cn/operation/Configuration.html' | relativize_url }})。
+通过Alluxio配置中的Alluxio-site.properties来配置Alluxio存储。 详细信息见
+[configuration settings]({{ '/en/operation/Configuration.html' | relativize_url}})。
 
-默认情况的一个常见修改是显式设置ramdisk的大小。例如，设置每个worker的ramdisk大小为16GB：
+对默认值的常见修改是明确设置ramdisk的大小。 例如，设置
+每个worker的ramdisk大小为16GB：
 
-```
+```properties
 alluxio.worker.memory.size=16GB
 ```
 
-另一个常见设置是指定多个存储介质，如ramdisk和SSD。 我们需要更新`alluxio.worker.tieredstore.level0.dirs.path`来指定我们想要的每个存储介质作为存储目录。例如，要使用ramdisk（安装在`/mnt/ramdisk`）和两个
-SSD（安装在`/mnt/ssd1`和`/mnt/ssd2`）：
+另一个常见更改是指定多个存储介质，例如ramdisk和SSD。 需要
+更新`alluxio.worker.tieredstore.level0.dirs.path`以指定想用的每个存储介质
+为一个相应的存储目录。 例如，要使用ramdisk(挂载在`/mnt/ramdisk`上)和两个
+SSD(挂载在`/mnt/ssd1`和`/mnt/ssd2`):
 
-```
+```properties
 alluxio.worker.tieredstore.level0.dirs.path=/mnt/ramdisk,/mnt/ssd1,/mnt/ssd2
+alluxio.worker.tieredstore.level0.dirs.mediumtype=MEM,SSD,SSD
 ```
 
-提供的路径应该指向本地文件系统中安装的相应存储介质的路径。为了启动短路操作，这些路径的权限应该允许客户端用户在改路径上读写和执行。例如，与启动Alluxio服务的用户在同一用户组的客户端用户需要770的权限。
+请注意，介质类型的顺序必须与路径的顺序相符。
+MEM和SSD是Alluxio中的两种预配置存储类型。
+`alluxio.master.tieredstore.global.mediumtype`是包含
+所有可用的介质类型的配置参数，默认情况下设置为`MEM，SSD，HDD`。 如果用户有额外存储介质类型可以通过修改这个配置来增加。
 
-更新存储介质后，我们需要指出为每个存储目录分配多少存储空间。例如，如果我们想在ramdisk上使用16 GB，在每个SSD上使用100 GB，
+提供的路径应指向挂载适当存储介质的本地文件系统中的路径。 
+为了实现短路操作，对于这些路径，应允许客户端用户在这些路径上进行读取，写入和执行。 
+例如，对于与启动Alluxio服务的用户组同组用户应给予`770`权限。
 
-```
+更新存储介质后，需要指出每个存储目录分配了多少存储空间。 
+例如，如果要在ramdisk上使用16 GB，在每个SSD上使用100 GB：
+
+```properties
 alluxio.worker.tieredstore.level0.dirs.quota=16GB,100GB,100GB
 ```
 
-请注意配额的序列必须与路径的序列相匹配。
+注意存储空间配置的顺序一定与存储目录的配置相符。
+
+`alluxio.worker.memory.size`和`alluxio.worker.tieredstore.level0.dirs.quota`之间有细微差别，默认配置为前者。Alluxio在通过`Mount`或`SudoMount`选项启动时，配置并挂载ramdisk。无论`alluxio.worker.tieredstore.level0.dirs.quota` 设置为何值，这个ramdisk大小是由`alluxio.worker.memory.size`确定的。 同样，如果要使用除默认Alluxio发放的ramdisk以外的设备，应该独立于内存大小来设置quota配置。
+
+###多层存储
```

Comment:
space after ### 
One kind of markdown code style, otherwise the line may not be processed as the heading format